frontend/libs/health-department/feature-case-detail/src/lib/diary-entries-list-item/diary-entries-list-item.component.ts

```diff
@@ -0,0 +1,47 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+export interface TrackedCaseDiaryEntry {
+  date: string;
+  morning: TrackedCaseDiaryEntrySlot;
+  evening: TrackedCaseDiaryEntrySlot;
+}
+
+interface TrackedCaseDiaryEntrySlot {
+  bodyTemperature: number;
+  symptoms: TrackedCaseDiaryEntrySymptom[]
+  contacts: TrackedCaseDiaryEntryContact[];
+}
+
+interface TrackedCaseDiaryEntrySymptom {
+  name: string;
+}
+
+interface TrackedCaseDiaryEntryContact {
+  firstName: string;
+  lastName: string;
+}
```

Comment:
Die Models sind doch schon in frontend/libs/health-department/domain/src/lib/model/tracked-cases-diary-entry.ts definiert?